.github/SECURITY.md

```diff
@@ -4,60 +4,106 @@
+| 12             | :large_orange_diamond: |             |
+| 11             | :large_orange_diamond: |             |
 
-Versions of CLang:
+| macOS |         Status         |     Recommended      |
+| :---- | :--------------------: | :------------------: |
+| 14    |   :white_check_mark:   | :large_blue_diamond: |
+| 13    | :large_orange_diamond: |                      |
+| 12 ≤  |      :red_circle:      |                      |
 
-| CLang Version | Supported          |
-| ------------- | ------------------ |
-| 18            | :white_check_mark: |
-| 15            | :white_check_mark: |
-| 14 and lower  | :red_circle:       |
+| Windows       |       Status       | Recommended |
+| :------------ | :----------------: | :---------: |
+| Windows 10    | :white_check_mark: |
+| Windows 11    | :white_check_mark: |
+| Windows 8.1 ≤ |    :red_circle:    |
 
-Versions of GCC:
+</details>
+<br>
 
-| GCC Version  | Supported          |
-| ------------ | ------------------ |
-| 14           | :white_check_mark: |
-| 12           | :white_check_mark: |
-| 11 and lower | :red_circle:       |
+### Supported Boost Versions:
 
-Versions of Ubuntu:
+| Boost  |       Status       |     Recommended      |
+| :----- | :----------------: | :------------------: |
+| 1.70 ≥ | :white_check_mark: | :large_blue_diamond: |
 
-| Ubuntu version  | Supported          |
-| --------------- | ------------------ |
-| 24.04           | :white_check_mark: |
-| 22.04           | :white_check_mark: |
-| 20.04 and lower | :red_circle:       |
+### Supported OpenSSL Versions:
 
-Versions of macOS:
+| OpenSSL |       Status       |     Recommended      |
+| :------ | :----------------: | :------------------: |
+| 3.X.X ≥ | :white_check_mark: | :large_blue_diamond: |
 
-| macOS Version | Supported          |
-| ------------- | ------------------ |
-| 14            | :white_check_mark: |
-| 12 and lower  | :red_circle:       |
+### Supported CMake Versions:
+
+| CMake  |       Status       |     Recommended      |
+| :----- | :----------------: | :------------------: |
+| 3.16 ≥ | :white_check_mark: | :large_blue_diamond: |
+
+### Supported MySQL Versions:
+
+| MySQL |       Status       |     Recommended      |
+| :---- | :----------------: | :------------------: |
+| 8.4 ≥ | :white_check_mark: | :large_blue_diamond: |
+| 8.0   | :white_check_mark: |                      |
+| 8.1   |    :red_circle:    |                      |
+| 8.0 < |    :red_circle:    |                      |
+
+### Supported CLang Versions:
+
+| CLang |       Status       | Recommended |
+| :---- | :----------------: | :---------: |
+| 18    | :white_check_mark: |             |
+| 15    | :white_check_mark: |             |
+| 14 ≤  |    :red_circle:    |             |
+
+### Supported GCC Versions:
+
+| GCC   |       Status       |     Recommended      |
+| :---- | :----------------: | :------------------: |
+| 8.4 ≥ | :white_check_mark: | :large_blue_diamond: |
```

Comment:
This looks incirrect 👀

Reply:
I'm on the phone right now, what's wrong there, the version?

Reply:
I see now, will fix that in a bit